Homework7/Fighter.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+
+namespace Homework7
+{
+    public class Fighter
```

Comment:
`FighterFactory` or something like this.

Reply:
fixed